package.json

```diff
@@ -16,26 +16,27 @@
     "LICENSE"
   ],
   "scripts": {
-    "test": "jshint lib index.js && jscs lib index.js && mocha -t 5000 -b -R spec test/index",
-    "cover": "nyc --reporter=lcov --reporter=text-summary npm test"
+    "lint": "eslint .",
+    "test": "jshint lib index.js && mocha -t 5000 -b -R spec test/index",
+    "cover": "eslint . && nyc --reporter=lcov --reporter=text-summary npm test"
```

Comment:
if linting is done as a `pretest` it won't need to exist as part of this command.